docs/en/Debugging-Guide.md

```diff
@@ -58,6 +59,48 @@ bucket, without the `s3://`, `s3a://`, or `s3n://` prefix.
 
 ## Usage FAQ
 
+#### Q: Why I see exceptions like "No FileSystem for scheme: alluxio"
+
+A: This error message is seen when your applications (e.g., MapReduce, Spark and etc) tried to access
+Alluxio as an HDFS-compatible data service, but the `alluxio://` scheme is not configured yet and thus not recognized by the
+application. Please make sure your HDFS configuration file `core-site.xml` (in your default hadoop
+installation or `spark/conf/` if you customize this file for Spark) has the following content:
+
+```xml
+<configuration>
+  <property>
+    <name>fs.alluxio.impl</name>
+    <value>alluxio.hadoop.FileSystem</value>
+  </property>
+</configuration>
+```
+
+#### Q: Why I see exceptions like "java.lang.RuntimeException: java.lang.ClassNotFoundException: Class alluxio.hadoop.FileSystem not found"
+
+A: This error message is seen when your applications (e.g., MapReduce, Spark and etc) tried to access
+Alluxio as an HDFS-compatible data service, the `alluxio://` scheme has been
+configured correctly but the Alluxio client jar is not found on the classpath of your application.
+
+- For MapReduce users, you can append the client jar to `$HADOOP_CLASSPATH`
```

Comment:
since Alluxio 1.3,  passing MapReduce applications with`libjars` option has an issue in loading redundant class loaders and causing problems, to avoid this happening, we anyway need to set `$HADOOP_CLASSPATH` I intentionally